hw-probe.pl

```diff
@@ -14433,22 +14433,21 @@ ()
                 $Name = "ghostbsd";
             }
             
-            my $GhostBSDPortsVer = "";
+            my $GhostBSDVer = "";
             if($Opt{"FixProbe"}) {
-                $GhostBSDPortsVer = readFile($FixProbe_Logs."/ports_ver");
+                $GhostBSDVer = readFile($FixProbe_Logs."/ghostbsd-version");
             }
             else
             {
-                if(checkCmd("pkg"))
+                if(checkCmd("ghostbsd-version"))
                 {
-                    $GhostBSDPortsVer = runCmd("pkg rquery \'\%v\' ports");
```

Comment:
Please preserve backward compatibility with old ghostbsd versions as well.

Reply:
The reason I did not keep the backward compatibility is that GhostBSD is a rolling release and if a user want to install hw-probe they have no choice to upgrade their system to the latest. We are not on a traditional release like FreeBSD or Ubuntu.

GhostBSD package manager will complain that the system needs an upgrade before installing the package.

All that said I can put the backward compatibility with no problem.

Reply:
Sorry for late reply. This is for compatibility with old GhostBSD installations. It should fallback to old version of this code if `ghostbsd-version` is not found. People may install the tool w/o using the package manager.

Reply:
the old way to find version is back.